system/http.go

```diff
@@ -40,6 +41,7 @@ func NewDefHTTP(httpStr string, system *System, config util.Config) HTTP {
 		str := strings.SplitN(r, ": ", 2)
 		headers.Add(str[0], str[1])
 	}
+	fmt.Printf("\nheaders: %v\n\n", headers)
```

Comment:
probably not needed

Reply:
yup, leftover from debugging, will remove